lib/generators/ajaxful_rating/templates/star_controller.js

```diff
@@ -1,25 +1,25 @@
-// Adapted from: https://dev.to/tonyrowan/adding-a-star-rating-with-hotwire-4p02
+// From: https://dev.to/tonyrowan/adding-a-star-rating-with-hotwire-4p02
 // Copied from: https://github.com/denkungsart/ajaxful-rating
 import { Controller } from "@hotwired/stimulus"
 
 export default class extends Controller {
-    static targets = ["star"];
+	static targets = ["star"];
```

Comment:
I think this got accidentially converted to tabs - let's stick to our defaults 🙈

Reply:
I could have sworn I just fixed this! How have the tabs gotten larger. sigh.

Reply:
I figured this out and will fix it.